cEP-0010.md

```diff
@@ -136,10 +138,37 @@ Here is the list of aspects to be implemented along with their corresponding
 
 - `Root.Formatting`: This aspect checks on the format (the structure) of your
   codebase.
-- `Root.Formatting.Quotation`: This aspect checks on the quotations being used
-  in your files given `Root.Formatting.Quotation.preferred_quotation`.
+- `Root.Formatting.Quotation`: This aspect represents the quotation marks
+  being used in the files written in languages with multiple string delimiters,
```

Comment:
"with multiple string delimiters" isnt quite correct as some tastes are about how to handle quotation when there are collisions.

Also, "string" -> "string literal" as this aspect is not about string variables.

Reply:
"string" -> "string literal" is fine but are you sure that "with multiple string delimiters" is not correct. I wrote it so as the tastes which we are using for escape characters are used just in order to maintain quotation consistency.

Reply:
hmmm ... I am not happy with the phase "with multiple string delimiters", but maybe it isnt wrong.

There are two ways 'quotation' might be seen --

1. quote characters encompassing a string literal, or
2. the process of encompassing a string literal.

I suspect the first of those means the aspect is not very useful as it applies only to a few languages which have multiple quote characters.

I'm going to think about this one some more.